makefile

```diff
@@ -12,6 +12,7 @@ pretty: node_modules
 	$(PRETTIER) --write src/main/webapp/css/*.css
 	find src/main/java -iname *.java | xargs $(CLANG_FORMAT) -i
 	find src/main/webapp -iname *.js | xargs $(CLANG_FORMAT) -i
+	find src/test/java -iname *.java | xargs $(CLANG_FORMAT) -i
```

Comment:
With `find`, it's possible to specify multiple directories to avoid repeating the same command.

`find src/main/java src/test/java -iname *.java | xargs $(CLANG_FORMAT) -i`